api/api/spec/spec.yaml

```diff
@@ -2726,8 +2844,254 @@ paths:
                     id: 1003
                     policy_id: "system_audit"
                     compliance:
-                    - 
+  /decoders:
+    get:
+      tags:
+        - decoders
+      summary: 'Get all decoders'
+      description: "Returns information about all decoders included in ossec.conf. This information include decoder's route, decoder's name, decoder's file among others"
+      operationId: api.controllers.decoders_controller.get_decoders
+      parameters:
+        - $ref: '#/components/parameters/pretty'
+        - $ref: '#/components/parameters/wait_for_complete'
+        - $ref: '#/components/parameters/offset'
+        - $ref: '#/components/parameters/limit'
+        - $ref: '#/components/parameters/sort'
+        - $ref: '#/components/parameters/search'
+        - $ref: '#/components/parameters/file'
+        - $ref: '#/components/parameters/pathParam'
+        - $ref: '#/components/parameters/status'
+
+      responses:
+        '200':
+          description: 'List of decoders included in ossec.conf'
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/ApiResponse'
+                - type: object
+                  properties:
+                    data:
+                      allOf:
+                        - $ref: '#/components/schemas/ListMetadata'
+                        - type: object
+                          properties:
+                            items:
+                              type: array
+                              items:
+                                $ref: '#/components/schemas/Decoder'
+              example:
+                error: 0
+                data:
+                  totalItems: 571
+                  items:
+                  - file: 0005-wazuh_decoders.xml
+                    path: /var/ossec/ruleset/decoders
```

Comment:
Absolute paths cannot be returned to user. Please, modify this example to be realistic.

Reply:
Ok, I going to modify all return which include paths. Thanks.